PyRDF/Proxy.py

```diff
@@ -25,6 +25,11 @@ def _call_handler(self, *args, **kwargs):
 
         if not self.action_node.value:
             generator = CallableGenerator(self.action_node._get_head())
+
+            if not Proxy.backend:
```

Comment:
To make `Proxy` completely agnostic about the `backend` module, I would move the setting of the default backend to here:
https://github.com/shravan97/PyTDF/blob/186df28db5a2a8bbe20f94c58858c5f8c1af351c/PyRDF/__init__.py#L7

Just by adding this:
```python
# Set default backend
Proxy.backend = Local()
```